index.html

```diff
@@ -1122,6 +1122,13 @@ <h3>Workflows and Exchanges</h3>
       </p>
       <span class="issue">These examples will be added later.</span>
       <p>
+VC API exchanges are expected to be short-lived but they can be combined with other mechanisms to enable longer interactions.
+Examples of other mechanisms include SMS, email, web notifications, or phone calls.
+This approach simplifies digital wallet implementation.
```

Comment:
```suggestion
This approach simplifies digital wallet implementation and allows
the reuse of existing mechanisms without reinvention within
the VC API. The Web or native platforms are expected to enable
additional interactions via applications (such as Web browsers)
or other platform features.
```

Reply:
Tweak to @dlongley's suggestion
```suggestion
This approach simplifies digital wallet implementation and allows
existing mechanisms to be reused without reinvention within
the VC API. The Web or native platforms are expected to enable
additional interactions via applications (such as Web browsers)
or other platform features.
```